CONTRIBUTION.md

```diff
@@ -0,0 +1,161 @@
+Table of Contents
```

Comment:
Just out of curiosity, is this generated by a tool?

Reply:
I used this script -  https://github.com/ekalinin/github-markdown-toc